carla/data/api/data.py

```diff
@@ -93,6 +93,23 @@ def normalized(self):
         """
         pass
 
+    @abstractmethod
+    def set_normalized(self, mlmodel):
```

Comment:
Unsure if its a good idea to make the Data class dependent on the Model class and vice versa. Also if we have the normalization (pipeline) encapsulated in the MlModel class, do we still need this here? If not then we could remove the encoded, normalized, encoded_normalized properties from the DataCatalog.

If we really need it then i'd rather make this function dependend on the scaler, rather the mlmodel

Reply:
The good thing about these properties from Data is, that we only need to encode and scale the data once, but this would only be for convenience 😄 
I like both ideas, loosing the dependencies, and the convenience of already encoden and normalized data

Reply:
Lets get rid of the dependency then first and if we see later indeed need it for convinience (and maybe have a less coupled design then). WDYT?

Reply:
Sounds good, I'll add these changes with the next commit 👍

Reply:
@Philoso-Fish I'd say for now we delete these properties. We only need encoded/normalized data for the predictions right now and there we make use of the MLModel properties. If we have a need for centrally storing them later lets thing of something then 👍🏽

Reply:
Done 👍